server/routerlicious/packages/services-utils/src/morganLoggerMiddleware.ts

```diff
@@ -46,7 +46,7 @@ export function jsonMorganLoggerMiddleware(
             }
             const properties = {
                 [HttpProperties.method]: tokens.method(req, res),
-                [HttpProperties.pathCategory]: `${req.baseUrl}${req.route ? req.route.path : "PATH_UNAVAILABLE"}`,
+                [HttpProperties.pathCategory]: `${request.baseUrl}${request?.route?.path ?? "PATH_UNAVAILABLE"}`,
```

Comment:
Should there also be a `?` after the first `request` here? Or was the `?` on the 2nd one added by mistake?

Reply:
Good point! I think it's not needed, since req and res are provided by express (morgan?) and I believe they would not be undefined.

Your comment also helped me realize I made a big typo which I didn't catch in my local testing (I should have used `req` instead of `request`). Will send a new iteration soon :)